src/azure-cli/azure/cli/command_modules/acs/_helpers.py

```diff
@@ -31,6 +36,14 @@ def _set_vm_set_type(vm_set_type, kubernetes_version):
             not specified and kubernetes version(%s) less than 1.12.9 only supports \
             availabilityset\n' % (kubernetes_version))
             vm_set_type = "AvailabilitySet"
-        else:
-            vm_set_type = "VirtualMachineScaleSets"
-    return vm_set_type.capitalize()
+
+    if not vm_set_type:
```

Comment:
Why add this again? Its already checked on L28. 
You can just keep L34, L35 as is and update L36 with the logic below from L43-L49

Reply:
Something like 
` else:
            vm_set_type = "VirtualMachineScaleSets"

 # normalize as server validation is case-sensitive
    if vm_set_type.lower() == "AvailabilitySet".lower():
        vm_set_type = "AvailabilitySet"

    if vm_set_type.lower() == "VirtualMachineScaleSets".lower():
        vm_set_type = "VirtualMachineScaleSets"
    return vm_set_type`

Reply:
I just reverted the logic to the original logic that Tony had for SLB. I shouldn't have refactored it in my previous PR (unrelated to IPWL). 

Going to leave it as Tony's logic - we can consider updating it in a separate PR if you want

Reply:
I see. Sounds good.